include/pybind11/detail/class.h

```diff
@@ -312,7 +312,31 @@ inline void traverse_offset_bases(void *valueptr,
     }
 }
 
+#ifdef Py_GIL_DISABLED
+static inline void enable_try_inc_ref(PyObject *op) {
+    // TODO: Replace with PyUnstable_Object_EnableTryIncRef when available.
+    // See https://github.com/python/cpython/issues/128844
+    if (_Py_IsImmortal(op)) {
```

Comment:
Would it make sense to rename `op` to `obj`? (Or are there any special requirements for `op` that you want to reflect with the variable name?)

Reply:
No special reason. `op` is just commonly used as a variable name for Python objects in CPython so that's what I'm used to now.